package.json

```diff
@@ -167,6 +167,16 @@
 					}
 				}
 			}
+		],
+		"keybindings": [
+			{
+				"command": "meadow.refreshDeviceList",
+				"when": "editorTextFocus",
+				"mac": "cmd+shift+d",
```

Comment:
As much as I really want to have the same command across platforms, this is currently the shortcut for opening the Debug view (**Show Run and Debug** command). We might need a different one for VS Code use to avoid stomping on that [default](https://code.visualstudio.com/docs/getstarted/keybindings#_display).

![VS Code command picker with the View command Show Run and Debug highlighted with a shortcut of command, shift, D.](https://user-images.githubusercontent.com/713665/178156040-63cb58f3-a206-4486-9efa-7d113253743a.png)

Reply:
@patridge I rarely use VSCode in anger. As it sounds like you use it regularly, I'm open to any suggestions or just add a commit to what you think wouldn't clash.

Reply:
@patridge as this is a shelved PR/branch for future reference, the shortcut discussion should probably happen over on - https://github.com/WildernessLabs/VSCode_Meadow_Extension/pull/18 as that is the PR that is likely to go in before v1.0